File: mobile-ios/Models/Workflow.swift
import Foundation

struct WorkflowItem: Identifiable, Codable {
    let id: String
    let name: String
    let description: String
    let triggerEvent: String
    let actionType: String
    let status: String
    let totalExecutions: Int
}

struct SystemAnalytics: Codable {
    let activeWorkflowsCount: Int
    let totalExecutionsCount: Int
    let anomalyCount: Int
    let averageExecutionTimeMs: Double
    let systemHealthScore: Double
}

File: mobile-android/app/src/main/java/com/nexusops/model/Workflow.kt
package com.nexusops.model

data class WorkflowItem(
    val id: String,
    val name: String,
    val description: String,
    val triggerEvent: String,
    val actionType: String,
    val status: String,
    val totalExecutions: Int
)

data class SystemAnalytics(
    val activeWorkflowsCount: Int,
    val totalExecutionsCount: Int,
    val anomalyCount: Int,
    val averageExecutionTimeMs: Double,
    val systemHealthScore: Double
)
